valqueries-automapper/src/main/java/com/valqueries/automapper/SqlGenerator.java

```diff
@@ -17,10 +18,10 @@
 
 public class SqlGenerator {
 	private static Logger logger = LoggerFactory.getLogger(SqlGenerator.class);
-	private SqlNameFormatter sqlNameFormatter;
-	private SqlDialect dialect;
-	private SqlDescriber sqlDescriber;
-	private Provider<ValqueriesSchemaBuilder> schemaBuilderProvider;
+	private final SqlNameFormatter sqlNameFormatter;
+	private final SqlDialect dialect;
+	private final SqlDescriber sqlDescriber;
+	private final Provider<ValqueriesSchemaBuilder> schemaBuilderProvider;
 
 	@Inject
 	public SqlGenerator(SqlNameFormatter sqlNameFormatter, DialectFactory dialectFactory, Database database, SqlDescriber sqlDescriber, Provider<ValqueriesSchemaBuilder> schemaBuilderProvider) {
```

Comment:
I don't understand the responsibility of the SqlGenerator does it only generate sql or also execute it?

Reply:
It does both. The class and method names are not great at explaining that.